eq/server/localServer.cpp

```diff
@@ -87,8 +87,11 @@ bool startLocalServer( const std::string& config )
     eq::server::Loader loader;
     eq::server::ServerPtr server;
 
-    if( config.length() > 3 && config.find( ".eqc" ) == config.length() - 4 )
+    if( config.length() > 3 &&
+        config.compare( config.size() - 4, 4, ".eqc" ) == 0 )
```

Comment:
why so much filtering here? The parser will fail if it doesn't find a eqc file header, so I would just pass the config string directly to the loader.